src/main/common/interest.ts

```diff
@@ -15,17 +16,13 @@ export async function getInterestDetails (claim: Claim): Promise<object> {
   if (claim.claimData.interestDate.type === InterestDateType.CUSTOM) {
     interestDate = claim.claimData.interestDate.date
   } else {
-    interestDate = claim.createdAt
+    interestDate = claim.issuedOn.startOf('day')
   }
 
-  const todayDate: Moment = MomentFactory.currentDate()
-  const noOfDays: number = todayDate.diff(interestDate, 'days') + 1
+  const todayDate: Moment = MomentFactory.currentDate().startOf('day')
```

Comment:
Definitely not needed, `#currentDate` factory already trims time by doing

```return moment().hours(0).minutes(0).seconds(0).milliseconds(0)```

Reply:
could update currentDate to use start of? it's cleaner than the current implementation?